cloud/amazon/route53_zone.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module: route53_zone
+short_description: add or delete Route53 zones
+description:
+    - Creates and deletes Route53 private and public zones
+options:
+    zone:
+        description:
+            - The DNS zone record (eg: foo.com.)
+        required: true
+        default: null
+    command:
```

Comment:
we do have a few modules that departed from the pattern in the past, confused a lot of people, that is why we are being stricter about it now